sdk/core/azure-core-http-netty/CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## 1.13.0-beta.1 (Unreleased)
 
 ### Features Added
+- Added specialized consumption for `HttpRequest.getBodyAsBinaryData()`.
```

Comment:
Do we need to call this out given it's an implementation detail?

Reply:
might be worth it if somebody attempts to use old core with new package. But I'll move it to "Others" section (like we did with clientlogger changes).